goconvey.go

```diff
@@ -45,15 +47,66 @@ func flags() {
 	flag.StringVar(&excludedDirs, "excludedDirs", "vendor,node_modules", "A comma separated list of directories that will be excluded from being watched.")
 	flag.StringVar(&workDir, "workDir", "", "set goconvey working directory (default current directory).")
 	flag.BoolVar(&autoLaunchBrowser, "launchBrowser", true, "toggle auto launching of browser.")
+	flag.BoolVar(&cleanupTmpFolder, "cleanupTemp", false, "cleanup tmp folder with reports and static web files")
 
 	log.SetOutput(os.Stdout)
 	log.SetFlags(log.LstdFlags | log.Lshortfile)
 }
 func folders() {
 	_, file, _, _ := runtime.Caller(0)
 	here := filepath.Dir(file)
-	static = filepath.Join(here, "/web/client")
-	reports = filepath.Join(static, "reports")
+	localStatic := filepath.Join(here, "web", "client")
+
+	currentUser, err := user.Current()
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	tmpDest := filepath.Join(os.TempDir(), fmt.Sprintf("/goconvey-%v", currentUser.Uid))
```

Comment:
Hm... since this is based on UID, what do you think about dropping a pidfile in this directory to do best-effort duplicate-goconvey detection? I suspect the UX for running two goconveys at the same time could be pretty bad, although I guess it's roughly the same badness as now...

Reply:
> pidfile in this directory 

I'll double check but the idea was to impl rewritting skipping in this specific case - that's should be enough, we're not modifying the static dir and reading from multiple goroutines should be good enough.

PID/lockfile should simplify this check, @riannucci thanks for the idea.

I'm expecting that reports will be generated from different packages when we're talking about multiple goconvey processes.

> I guess it's roughly the same badness as now...

It looks so, I might investigate this further, after this PR, next week.
Don't like how UI refreshes hang up from time to time too.